AsyncDisplayKit/Private/ASBatchFetching.m

```diff
@@ -61,9 +57,14 @@ BOOL ASDisplayShouldFetchBatchForContext(ASBatchContext *context,
 
   // target offset will always be 0 if the content size is smaller than the viewport
   BOOL hasSmallContent = offset == 0.0 && contentLength < viewLength;
+  if (hasSmallContent) {
```

Comment:
This is a descriptive variable name, I think it would be fine to use as return (hasSmallContent || (isScrollingTowardEnd && remaining <= trigger))